### PR TITLE
Parse artist role field

### PR DESCRIPTION
**Pull request by egh**
Very small change to parse the `role` field for an artist.

### CODE REVIEW

File: discogs_client/models.py [hunk 1]
```diff
@@ -478,7 +478,8 @@ class Artist(PrimaryAPIObject):
     #: single string, for example: "DJ ABC Feat MC Z". Also check out the
     #: ``artists_sort`` attribute of a ``Release`` object.
     join = SimpleField()
-
+    role = SimpleField()
```
Comment (AnssiAhola) [inline discogs_client/models.py:481]: Can you add a similar comment as the `join` field as this field seems to be populated only(?) when accessing the `artists` field of an `Release`/`Master Release` object.

https://www.discogs.com/developers#page:database,header:database-release-get
https://www.discogs.com/developers#page:database,header:database-master-release-get

**Review by AnssiAhola — COMMENTED** [review]
Hey @egh, thank for the PR.

I'll test this out over this weekend, but should be pretty straight forward.

Only thing being that this field is only populated when accessing the artist from a specific endpoint(s) as I mentioned in my review comment,
but shouldn't be an issue as the value will just be `None` if it isn't populated.

File: discogs_client/models.py (hunk 1 above)
Comment (egh) [reply discogs_client/models.py:483]: Done! Let me know if the comment is not sufficient. Thanks!

Comment (JOJ0) [reply discogs_client/models.py:483]: The wording is perfectly fine. The second line requires #: as the beginning of the line though.

Comment (egh) [reply discogs_client/models.py:483]: Thanks, fixed!

**Review by AnssiAhola — APPROVED** [review]
LGTM!
If there's no hurry to release this I would wait and see if any other PR comes up, before releasing a new version.
What do you think @JOJ0 ?

**Comment by JOJ0** [comment]
Thanks @AnssiAhola and @egh, I could do a release soonish - within this week? No worries, we have the tools ready 😀 

@egh please double check if our autogen docs render nicely. Latest on python3-discogs-client.readthedocs.io reflects master branch in git.

**Comment by JOJ0** [comment]
LGTM: https://python3-discogs-client.readthedocs.io/en/latest/discogs_client.html#discogs_client.models.Artist.role

**Comment by JOJ0** [comment]
https://pypi.org/project/python3-discogs-client/2.7/

Thanks again to you both! @egh @AnssiAhola

**Pull request merged**